solutions/usaco-738.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: usaco-738
+source: USACO
+title: Paired Up
+author: Sofia Yang
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_pairup_silver_open17.html)
+
+**Time Complexity:** $\mathcal{O}(N\log N)$
```

Comment:
maybe it's beneficial to add a short snippet on why this is n log n (sorting) since this differs from what the official analysis states.

Reply:
I'm sorry, did I get the complexity wrong?
In both my implementation and in the official editorial, it was necessary to sort the N cows, and sorting is approximately 
order N log N, so...